lib/internal/Magento/Framework/Pricing/PriceCurrencyInterface.php

```diff
@@ -79,9 +79,10 @@ public function convertAndFormat(
      * Round price
      *
      * @param float $price
+     * @param int $precision
      * @return float
      */
-    public function round($price);
+    public function round($price, $precision = self::DEFAULT_PRECISION);
```

Comment:
Changing the interface signature will lead to fatal error on instances where this interface is implemented. That can be resolved by introducing a new method implementing a new interface for PriceCurrency class